kotlinx.fuzz.jazzer/src/main/kotlin/kotlinx/fuzz/jazzer/JazzerEngine.kt

```diff
@@ -58,13 +61,15 @@ class JazzerEngine(private val config: KFuzzConfig) : KFuzzEngine {
         FuzzTargetRunner.registerFatalFindingHandlerForJUnit { bytes, finding ->
             atomicFinding.set(finding)
             val hash = MessageDigest.getInstance("SHA-1").digest(bytes).toHexString()
-            val file = File(Path(Opt.reproducerPath.get()).toAbsolutePath().toString(), "stacktrace-$hash")
-            file.createNewFile()
-            file.writeText(
-                finding.stackTraceToString().split("\n")
-                    .takeWhile { it.trim() != "at kotlinx.fuzz.jazzer.JazzerTarget.fuzzTargetOne(JazzerTarget.kt:17)" }
-                    .joinToString("\n")
-            )
+            val file = Paths.get(Opt.reproducerPath.get(), "stacktrace-$hash")
+            if (!file.exists()) {
+                file.createFile()
+                file.writeText(
+                    finding.stackTraceToString().split("\n")
+                        .takeWhile { it.trim() != "at kotlinx.fuzz.jazzer.JazzerTarget.fuzzTargetOne(JazzerTarget.kt:17)" }
```

Comment:
This `takeWhile` looks very unreliable. The `JazzerTarget` may change and this filtering will break. Could you filter stackTrace directly to overcome that?

Reply:
Fixed, but still changes (renaming method, e.g.) in `JazzerTarget` can lead to incorrect behaviour